### DISCUSSION TITLE
Windows Event logs

### DISCUSSION THREAD

**Discussion by hernanPSS** [wazuh/wazuh#15079] [Q&A]
I'm new to wazuh and may be I'm doing something wrong or I'm looking in the wrong place....

I want to look for Windows change password events in wazuh (Event ID 4723 or 4724) which I see in one of my DCs security Event Log but I cannot find them in the Security Events in the wazuh console.... 

I was looking in the local ossec.conf in the agent and this events don´t look to be filtered...

<localfile>
    <location>Security</location>
    <log_format>eventchannel</log_format>
    <query>Event/System[EventID != 5145 and EventID != 5156 and EventID != 5447 and
      EventID != 4656 and EventID != 4658 and EventID != 4663 and EventID != 4660 and
      EventID != 4670 and EventID != 4690 and EventID != 4703 and EventID != 4907 and
      EventID != 5152 and EventID != 5157]</query>
  </localfile>

any ideas???

thanks in advance!!!!

**Comment by mauceballos** [comment]
Hi hernanPSS

thanks for using Wazuh.
do you have configured or created rules in Wazuh? if not, you should.
please see this [documentation](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/ruleset.html?highlight=configuring%20rules) .

best

**Comment by zbalkan** [comment]
Hi @hernanPSS ,

The part you copy-pasted includes event log IDs that create a lot of noise. So they are basically ignored on the agent side.
```
Security eventchannel Event/System[EventID != 5145 and EventID != 5156 and EventID != 5447 and EventID != 4656 and EventID != 4658 and EventID != 4663 and EventID != 4660 and EventID != 4670 and EventID != 4690 and EventID != 4703 and EventID != 4907 and EventID != 5152 and EventID != 5157]
```

You can find Windows security related events in [0580-win-security_rules.xml](https://github.com/wazuh/wazuh/blob/master/ruleset/rules/0580-win-security_rules.xml). And as you can see, there is no rule for that. You can just create a custom one based on the [rule syntax documentation](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html). Please do not forget to set a severity that makes sense from 0 to 16. You can find what each severity level means in the [docs](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html) too.
